src/vm/dllimport.cpp

```diff
@@ -2255,28 +2255,9 @@ void NDirectStubLinker::DoNDirect(ILCodeStream *pcsEmit, DWORD dwStubFlags, Meth
 
 #ifndef CROSSGEN_COMPILE
```

Comment:
Nit: `#ifndef CROSSGEN_COMPILE` can be removed as well, I think.

Reply:
@jkotas Done